src/store/modules/platform/platform.ts

```diff
@@ -98,7 +102,15 @@ const platform_module: Module<PlatformState, RootState> = {
 
                 const delegatorsPending: DelegatorPendingRaw[] = delegatorPendingMap[nodeID] || []
 
-                const delegatedAmt = new BN(v.delegatorWeight)
+                let delegatedAmt = new BN(0)
```

Comment:
why this is 0 ?

Reply:
This is to initialize the value according to BN format